include/nbl/asset/CCPUMeshPackerV2.h

```diff
@@ -67,12 +68,12 @@ class CCPUMeshPackerV2 final : public IMeshPacker<ICPUMeshBuffer, MDIStructType>
     template <typename Iterator>
     bool commit(IMeshPackerBase::PackedMeshBufferData* pmbdOut, ReservedAllocationMeshBuffers* rambIn, const Iterator begin, const Iterator end);
 
-    PackerDataStore createGPUPackerDataStore(video::IVideoDriver* driver);
+    PackerDataStore<video::IGPUBuffer> createGPUPackerDataStore(video::IVideoDriver* driver);
```

Comment:
nah, you should have GPUMeshPacker and it should take driver in constructor, look at IGPUVirtualTexture